test/functional/change_stream_spec_tests.js

```diff
@@ -0,0 +1,202 @@
+'use strict';
+
+var setupDatabase = require('./shared').setupDatabase;
+const chai = require('chai');
+const fs = require('fs');
+const expect = chai.expect;
+const camelCase = require('lodash.camelcase');
+chai.use(require('chai-subset'));
+
+describe.only('Change Stream Spec', function() {
+  const specStr = fs.readFileSync(`${__dirname}/spec/change-stream/change-streams.json`, 'utf8');
+  const specData = JSON.parse(specStr);
+  const ALL_DBS = [specData.database_name, specData.database_name_2];
+
+  const delay = ms => new Promise(r => setTimeout(r, ms));
```

Comment:
this has _got_ to exist in other places, let's put it into a utilities file?

Reply:
it actually already exists in `./shared` 😊